### DISCUSSION TITLE
Rumble integration Broken after Rumble update.

### DISCUSSION THREAD

**Discussion by AlphaPyxis** [3dproger/AxelChat#253] [General]
Hey!

Just thought id make you aware that the new rumble UI update seems to have broken axel chat's ability to connect to rumble. It is stuck in a loop of "connecting" and doesn't connect. Hopefully it isnt something too serious.

Thank you in advance,

Alpha.

**Comment by 3dproger** [comment]
Hello!

Thanks for the prompt information! I just fixed it in update 0.30.0.

AxelChat developer.
